mache-rabbit/src/main/java/com/excelian/mache/events/integration/RabbitMQConfig.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Type for the Rabbit MQ config.
  */
-public class RabbitMqConfig {
+public class RabbitMQConfig {
 
```

Comment:
One thing to watch is how this behaves itself up at github once it's merged in, I had to do a manual change within the github gui